tracer/test/test-applications/integrations/Samples.AzureServiceBus/README.md

```diff
@@ -0,0 +1,186 @@
+
+## Test application behavior
+The test application consists of 5 separate test methods, which will be explained individually.
+
+### TestServiceBusProcessorAsync
+Source: [`RequestHelper.Processor.cs`](./RequestHelper.Processor.cs)
+API tested: ServiceBusProcessor
+
+The test method initializes a ServiceBusProcessor, opens a root span, sends one message
+
+### TestServiceBusSessionReceiverAsync
+> Source: [`RequestHelper.Session.cs`](./RequestHelper.Session.cs)
+
+### TestSenderSchedulingAsync
+> Source: [`RequestHelper.cs`](./RequestHelper.cs)
+
+### TestServiceBusReceiverIndividualMessageAsync
+> Source: [`RequestHelper.cs`](./RequestHelper.cs)
+
+### TestServiceBusReceiverBatchMessagesAsync
+> Source: [`RequestHelper.cs`](./RequestHelper.cs)
+
+
+## Resulting Traces
+### TestServiceBusProcessorAsync traces
+Trace #1
+
+```plaintext
+SendMessageToProcessorAsync
+-----------
+|          |
+Message    ServiceBusSender.Send
+|
+ServiceBusProcessor.ProcessMessage
+|
+ServiceBusReceiver.Complete
+```
+
+Trace #2
+
+```plaintext
+ServiceBusReceiver.Receive
+```
+
+### TestServiceBusSessionReceiverAsync
+
+Trace #1
+
+```plaintext
+FirstSessionId_Producer
+------------------------------------------------
+|          |                        |           |
+Message    ServiceBusSender.Send    Message     ServiceBusSender.Send
+```
+
+Trace #2
+
+```plaintext
+SecondSessionId_Producer
+-----------
+|          |
+Message    ServiceBusSender.Send
+```
+
+Trace #3
+
+```plaintext
+ProcessorSessionId_Producer
+-----------
+|          |
+Message    ServiceBusSender.Send
+|
+ServiceBusSessionProcessor.ProcessSessionMessage
+|
+ServiceBusReceiver.Complete
+```
+
+Traces #5-13
+
+- `ServiceBusReceiver.Receive`
+- 2x `ServiceBusReceiver.Complete`
+- 1x `ServiceBusSessionReceiver.RenewSessionLock`
+- 1x `ServiceBusSessionReceiver.SetSessionState`
+- 1x `ServiceBusSessionReceiver.GetSessionState`
+- `ServiceBusReceiver.Receive` (receive from the SessionProcessor)
+- `ServiceBusReceiver.Complete` (complete from the SessionProcessor)
+- `ServiceBusReceiver.Receive` (Extra span from processing loop???)
+
+### TestSenderSchedulingAsync
+
+Trace #1
+
+```plaintext
+TestSenderSchedulingAsync
+---------------------------------------------------------------------------------------------------------------------------------
+|          |                            |                          |          |          |          |                            |
+Message    ServiceBusSender.Schedule    ServiceBusSender.Cancel    Message    Message    Message    ServiceBusSender.Schedule    ServiceBusSender.Cancel
+```
+
+### TestServiceBusReceiverIndividualMessageAsync
+
+
+Trace #1
```

Comment:
Trace_Id 53

Reply:
Added this in 46542c7eccdecdbb612460fde446cbc4ce3c2cdf